service/gradle/dependencies.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
   }
 
   //Terra Landing Zone Service
-  implementation ('bio.terra:terra-landing-zone-service:0.0.8-SNAPSHOT') {
+  implementation ('bio.terra:terra-landing-zone-service:0.0.9-rt-toaz-163-SNAPSHOT') {
```

Comment:
Will change this after https://github.com/DataBiosphere/terra-landing-zone-service/pull/22 merges